saleor/account/i18n.py

```diff
@@ -183,9 +180,8 @@ def validate_address(self, data):
         try:
             data["country_code"] = data.get("country", "")
             if data["street_address_1"] or data["street_address_2"]:
-                data["street_address"] = "%s\n%s" % (
-                    data["street_address_1"],
-                    data["street_address_2"],
+                data["street_address"] = (
+                    f'{data["street_address_1"]}\n' f'{data["street_address_2"]}'
```

Comment:
This could be one string
```suggestion
                    f'{data["street_address_1"]}\n{data["street_address_2"]}'
```